<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_updateBalance_master</class>
 <widget class="QDialog" name="Dialog_updateBalance_master">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>211</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>379</width>
    <height>211</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>379</width>
    <height>211</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="le_target_user">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Логин пользователя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>10</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Изменение баланса&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_user_balance">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>109</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Баланс пользователя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update_balance">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
